### PR TITLE
API for fetching Approved Organisations for a Smart Contract

### PR DESCRIPTION
**Pull request by ShubhIO**
- Added Controller of fetching the list of Organizations that approved the chaincode
- Added Service class implementation

Signed-off-by: Shubhkarman Walia <shubhkarman.walia@Walmart.com>

### CODE REVIEW

File: src/main/java/hlf/java/rest/client/controller/FabricOperationsController.java [hunk 1]
```diff
@@ -29,4 +31,22 @@
             networkName, chaincodeOperations, operationsType),
         HttpStatus.OK);
   }
+
+  @GetMapping(value = "/chaincode/approved-organisations")
+  public ResponseEntity<Set<String>> getApprovedOrganisationListForSmartContract(
+      @RequestParam("network_name") @Validated String networkName,
+      @RequestParam("network_name") String chaincodeName,
```
Comment (arsulegai) [inline src/main/java/hlf/java/rest/client/controller/FabricOperationsController.java:38]: Please correct these parameters

File: src/main/java/hlf/java/rest/client/service/impl/ChaincodeOperationsServiceImpl.java [hunk 2]
```diff
@@ -80,6 +88,68 @@ public String performChaincodeOperation(
     }
   }
 
+  @Override
+  public Set<String> getApprovedOrganizations(
+      String networkName,
+      ChaincodeOperations chaincodeOperationsModel,
+      Optional<LifecycleChaincodeEndorsementPolicy> chaincodeEndorsementPolicyOptional,
+      Optional<ChaincodeCollectionConfiguration> chaincodeCollectionConfigurationOptional) {
+    HFClient client = HFClient.createNewInstance();
```
Comment (arsulegai) [inline src/main/java/hlf/java/rest/client/service/impl/ChaincodeOperationsServiceImpl.java:97]: [nitpick] Reuse the bean for HFClient

Comment (ShubhIO) [reply src/main/java/hlf/java/rest/client/service/impl/ChaincodeOperationsServiceImpl.java:97]: Sure

File: src/main/java/hlf/java/rest/client/controller/FabricOperationsController.java (hunk 1 above)
Comment (ShubhIO) [reply src/main/java/hlf/java/rest/client/controller/FabricOperationsController.java:38]: Sure...thanks

**Review by arsulegai — APPROVED** [review]
LGTM

**Pull request merged**